app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -267,70 +267,39 @@ function buildArticleHTML(article) {
         '</div></a>';
     }
 
-    var navigationLink = `
-      <a
-        href="${article.path}"
-        aria-labelledby="article-link-${article.id}"
-        class="crayons-story__hidden-navigation-link"
-      >
-        ${article.title}
-      </a>
-    `;
-
-    var articleElement =
-      '<article class="crayons-story" ' +
-      'data-article-path="' +
-      article.path +
-      'id="article-' +
-      article.id +
-      '" data-content-user-id="' +
-      article.user_id +
-      '">' +
-      navigationLink +
-      '<div role="presentation">\
-        ' +
-      videoHTML +
-      '\
-        <div class="crayons-story__body">\
-          <div class="crayons-story__top">\
-            ' +
-      meta +
-      '\
-          </div>\
-          <div class="crayons-story__indention">\
-            <h2 class="crayons-story__title"><a href="' +
-      article.path +
-      '" id="article-link-' +
-      article.id +
-      '">' +
-      filterXSS(article.title) +
-      '</a></h2>\
-            <div class="crayons-story__tags">' +
-      tagString +
-      '</div>\
-            ' +
-      searchSnippetHTML +
-      '\
-            <div class="crayons-story__bottom">\
-              <div class="crayons-story__details">' +
-      reactionsDisplay +
-      commentsDisplay +
-      '</div>\
+    return `<article class="crayons-story"
+      data-article-path="${article.path}"
```

Comment:
Looks like `eslint` or any other of the pre-commit hooks reorganized this a bit for me. 

Not sure if it's as readable like this compared to wrapping the entire string in parenthesis and having the string start at the next line (271), that way it's all at the same indentation.

But that's what the tool suggests so it's okay I guess 🤔